objects.cpp

```diff
@@ -29,6 +29,9 @@ class Digital : public Stocked {
     int deliver() override {
         return Stocked::deliver() / 2;
     }
+    Item* prepare(int country) override {
+        return new Digital(stk);
```

Comment:
@MCJOHN974 this is wrong. You should use "copy" constructor, like it works in `Tangible`